projects/packages/publicize/src/class-publicize-utils.php

```diff
@@ -109,4 +109,14 @@ public static function assert_is_wpcom( $method ) {
 			throw new \Exception( esc_html( "Method $method can only be called on WordPress.com." ) );
 		}
 	}
+
+	/**
+	 * Check if the new module endpoint is available in the used Jetpack version.
+	 *
+	 * @return bool
+	 */
+	public static function has_new_module_endpoint() {
```

Comment:
Do you think we should link to the code/PR for this? I'm not sure it's obvious what this is checking, and I can imagine we would forget!

Also, it's an addition to the existing settings endpoint, so we might want to change the name, although I'm not sure what to!

Reply:
I renamed it to `should_use_jetpack_module_endpoint`

Reply:
Cool. Do you think we should link to the code? https://github.com/Automattic/jetpack/pull/41461/files#diff-f8e5ef1115599de750b64143dd1901554254eddd95ab4371b6b6b3b2a5914224R638-R642 I think it might be worth documenting that we need the module status in the response, and that's why we're doing the version check.